docs/DeveloperGuide.md

```diff
@@ -473,6 +490,7 @@ Use case ends.
 * **Mainstream OS**: Windows, Linux, Unix, MacOS
 * **Order**: Agreement made by customers with user on delivery of product
 * **Private contact detail**: A contact detail that is not meant to be shared with others
+* **Status**: The current fulfilment condition of an order, such as completed or pending.
```

Comment:
```suggestion
* **Status**: The current fulfilment condition of the delivery of an order, namely completed or pending.
```

Reply:
Negligible, although Im not sure if "condition" is the right word, but not much difference I guess.